pkg/controller/kogitoinfra/kogitoinfra_controller.go

```diff
@@ -131,13 +131,14 @@ func (r *ReconcileKogitoInfra) getReconcileResultFor(err error, requeue bool) (r
 	}
 	// no requeue, no errors, stop reconciliation
 	if !requeue && err == nil {
+		log.Debug("No need reconciliation for KogitoInfra")
 		return reconcile.Result{RequeueAfter: 0, Requeue: false}, nil
 	}
 	// caller is asking for a reconciliation
 	if err == nil {
-		log.Info("Waiting for all resources to be created, scheduling reconciliation")
+		log.Infof("Waiting for all resources to be created, scheduling reconciliation. Scheduling reconciliation for %d", reconciliationStandardInterval)
```

Comment:
A nitpick, the reconciliation period is hard to be read from logs:
```
{"level":"info","T":"2020-11-04T17:20:37.769Z","logger":"kogitoinfra_controller","msg":"Waiting for all resources to be created, scheduling reconciliation: infinispan instance kogito-infinispan not ready. Waiting for Condition.Type == True. Scheduling reconciliation for 30000000000"}
```
Can it be converted to something easily readable?